src/main/java/com/yash/ExpenseClaims/Application.java

```diff
@@ -0,0 +1,16 @@
+package com.yash.ExpenseClaims;
+
+import org.springframework.boot.SpringApplication;
+import org.springframework.boot.autoconfigure.SpringBootApplication;
+
+import static org.springframework.boot.Banner.Mode.OFF;
+
+@SpringBootApplication
+public class Application {
+
+    public static void main(String[] args) {
+        SpringApplication springApplication = new SpringApplication(Application.class);
```

Comment:
Please use SpringApplicationBuilder and [here](https://docs.spring.io/spring-boot/docs/2.0.4.RELEASE/api/org/springframework/boot/builder/SpringApplicationBuilder.html) is the API referece